authors/apps/authentication/serializers.py

```diff
@@ -31,11 +36,10 @@ def create(self, validated_data):
 
 
 class LoginSerializer(serializers.Serializer):
-    email = serializers.CharField(max_length=255)
+    email = serializers.EmailField(max_length=255)
```

Comment:
:+1: This is awesome, at least now you can't just enter an invalid email and then have it pass through the DB query checks